<launch>
    <!--机器人模型-->
	<arg name="model" default="$(find xacro)/xacro --inorder '$(find oryxbot_description)/urdf/oryxbot_uarm.urdf.xacro'" />
	<param name="robot_description" command="$(arg model)" />

	<node
	name="joint_state_publisher"
	pkg="joint_state_publisher"
	type="joint_state_publisher" />
	<node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" >
		<param name="publish_frequency" type="double" value="50.0" />
		<param name="tf_prefix" type="string" value="" />
	</node>
	<node name="swiftpro_rviz_node" pkg="oryxbot_description" type="swiftpro_rviz_node" />


	<!-- rviz -->
	<node pkg="rviz" type="rviz" name="rviz" required="true" args="-d $(find reinovo_control)/rviz/reinovo.rviz" output="screen"/>

	<param name="other1" value="手臂"/>
	<param name="other2" value="自主充电"/>
	<param name="other3" value="其他"/>


    <!--json 节点-->
	<arg name="custom_param_file" default="$(find reinovo_control)/param/action.yaml"/>
	<arg name="pkg_path"  default=" $(find reinovo_control)"/>
	<node name="json_file" pkg="reinovo_control" type="json_file.py" output="screen" >
		<rosparam file="$(arg custom_param_file)" command="load" />
		<param name="map_file" value="$(arg pkg_path)"/>
	</node>

</launch>
